src/amp/components/Epic.tsx

```diff
@@ -0,0 +1,168 @@
+import React from 'react';
+import { palette } from '@guardian/src-foundations';
+import { headline, body, textSans } from '@guardian/src-foundations/typography';
+import {css} from "emotion";
```

Comment:
prettier issue again, also its convention to split 3rd party lib and internal imports into 2 distinct sections

Reply:
Please see response on previous review 🙂